### DISCUSSION TITLE
使用Python分析旋轉的影像

### DISCUSSION THREAD

**Discussion by enoch-lightning** [1101-GS4719-C/Python-Community-GS4719-C#369] [General]
如果有一張圖片，經過旋轉，我想要知道其旋轉的角度，可以用很多種方式求解。目前在網路上看到可以使用像素比對、orb、surf、AOI瑕疵檢驗與質心座標(com, C.M.)來分析。

**Comment by enoch-lightning** [comment]
舉個旋轉的圖片例子如下，目前整個是一張圖片，我不知道是否需要先做什麼處理，才能做旋轉角度分析：
![旋轉影像實例_1](https://user-images.githubusercontent.com/86505454/153988747-d2fe5685-04e8-4c9e-aabd-1a9b1141f726.jpg)

**Comment by enoch-lightning** [comment]
這邊附上資料來源：
[https://www.facebook.com/photo/?fbid=957729854903604&set=gm.10161902818568438](https://www.facebook.com/photo/?fbid=957729854903604&set=gm.10161902818568438)

**Comment by pekoranoinu** [comment]
太強了!感謝大佬!
